logrotate/test/FilterTest.cpp

```diff
@@ -16,6 +16,13 @@
 #include <g3log/std2_make_unique.hpp>
 #include "RotateTestHelper.h"
 
+#if (defined(WIN32) || defined(_WIN32) || defined(__WIN32__)) && !defined(__MINGW32__)
+#define F_OK 0
```

Comment:
F_OK seems to be defined in NIX but not in windows. The equivalent according to 
https://msdn.microsoft.com/en-us/library/1w06ktdy.aspx
would be 
mode value | Checks file for
00               | Existence only